Go/syntax_test_go.go

```diff
@@ -35,22 +34,30 @@ You may have to disable Go-specific linters when working on this file.
 //                            ^^^^^^^^^^^ comment.line.go
 
     /*
+//  ^^ comment.block.go punctuation.definition.comment.begin.go
 // ^ -comment
 //  ^^^^ comment.block.go
     comment
 //  ^^^^^^^^ comment.block.go
     */
-//  ^^ comment.block.go
+//  ^^ comment.block.go punctuation.definition.comment.end.go
```

Comment:
The counter part for checking boundaries of the previous suggestion.

```suggestion
//^^ comment.block.go - punctuation
//  ^^ comment.block.go punctuation.definition.comment.end.go
```